website/docs/general/integrate-with-project-docs.md

```diff
@@ -0,0 +1,80 @@
+---
+id: integrate-with-project-docs
+title: Integrate with Project documentations
+---
+
+## docs directory structure
+
+Put all Markdown files into the latest directory
+
+```
+/docs
+├── assets  # 存放静态文件
+│   ├── images
+│   │   ├── xxxxx.png
+│   └── other
+│       └── xxxxx.xxx
+├── en
+│   └── latest
+│       ├── doc1.md
+│       ├── folder
+│       │   └── doc2.md
+│       │       └── folder2
+│       │           └── doc3.md
+│       └── config.json
+├── es
+│   └── latest
+│       └── ...
+└── zh
+    └── latest
+        └── ...
+```
+
+## Configuration file
+
+This file is located in /docs/<locale>/latest/config.json, where locale is locale codes, please refer to https://www.science.co.il/language/Locale-codes.php, locale is all lowercase. Don't forget to remove the comment after copying JSON.
```

Comment:
Why not wrap /docs/<locale>/latest/config.json with ``?

Reply:
done